### DISCUSSION TITLE
I introduced alphatab in npm in vue, and could not listen to events such as playerFinished and playerPositionChanged, and did not report errors. How can I introduce alphatab in vue to use it correctly? This problem has puzzled me for several days. Can you help me

### DISCUSSION THREAD

**Discussion by hhly** [CoderLine/alphaTab#991] [Q&A]
I introduced alphatab in npm in vue, and could not listen to events such as playerFinished and playerPositionChanged, and did not report errors. How can I introduce alphatab in vue to use it correctly? This problem has puzzled me for several days. Can you help me
<img width="670" alt="image" src="https://user-images.githubusercontent.com/29226405/193178427-12379182-50d5-4281-be83-22073a6a7781.png"> 
The time cannot be loaded, cannot be played, and the enablePlayer and enableCursor settings are invalid

**Comment by Danielku15** [comment]
Without a concrete example or some code it will be hard to give any good advise. Generally I would advise to set the [`logLevel`](https://alphatab.net/docs/reference/settings/core/loglevel) to `debug` to get really all log outputs and double check you have really all settings correctly filled. You could also check step-by-step what is different between your setup and the [tutorial](https://alphatab.net/docs/tutorial-web/introduction).
